### PR TITLE
added fragment types generator

### PR DESCRIPTION
**Pull request by bandvov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,11 +123,197 @@ It correctly bundles React in production mode and optimizes the build for the be
...
+addTypename: true,
+fragmentMatcher
+})
+})`
+
+Now apollo client can read introspection result.
+
+#### Usage
+
+before using generator you should run backend server
```
Comment (Alan18081) [inline README.md:311]: Please, put link to the backend here

File: README.md [hunk 2]
```diff
@@ -107,11 +123,197 @@ It correctly bundles React in production mode and optimizes the build for the be
...
 
-### `npm run create <Component name>`
+open terminal
```
Comment (Alan18081) [inline README.md:313]: Please, define steps for the running generator as an ordered list

File: README.md (hunk 1 above)
Comment (bandvov) [reply README.md:311]: added

File: README.md [hunk 3]
```diff
@@ -107,11 +123,197 @@ It correctly bundles React in production mode and optimizes the build for the be
...
+`{ "scripts": { "generate": "graphql-codegen" } }`
+
+This allows us to use `npm run generate` or `yarn generate` and get the introspection result when we need to.
+
+You must run `npm run generate` to generate the introspection file every time you add new unions or intarfaces. This will awoid errors as our introspection will always be up to date.
+
+You should modify your apollo config file:
+
+`import ApolloClient, { gql } from 'apollo-boost';
+import fetch from 'unfetch';
```
[the remark this answers is not in the document]
Comment (bandvov) [reply README.md:282]: code removed

File: README.md (hunk 2 above)
Comment (bandvov) [reply README.md:313]: I guess it is not duplicated. above is installation and setting up files. this is usage

Comment (bandvov) [reply README.md]: removed

File: README.md [hunk 4]
```diff
@@ -276,44 +276,10 @@ This allows us to use `npm run generate` or `yarn generate` and get the introspe
...
-
-const client = new ApolloClient({
-uri: REACT_APP_API_URL,
-fetch,
-cache: new InMemoryCache({
-addTypename: true,
-fragmentMatcher
-})
-})`
+You should modify your apollo config file.
```
[the remark this answers is not in the document]
Comment (bandvov) [reply README.md:279]: modified is not name but content

**Pull request merged**